js/popup.js

```diff
@@ -0,0 +1,235 @@
+       <p class="project-intro">A daily selection of privately personalized reads;
+       no accounts or sign-ups required.
+       has been the industry's standard dummy text ever since the 1500s,
+       when an unknown printer took a standard dummy text.</p>
+     <ul class="tag-list">
+       <li class="tag">css</li>
+       <li><span class="divider"></span></li>
+       <li class="tag">html</li>
+       <li><span class="divider"></span></li>
+       <li class="tag">Bootsrap</li>
+       <li><span class="divider"></span></li>
+       <li class="tag">Ruby</li>
+    </ul>
+     <button type="button" class="btn" onclick="">See project</button>
+   </div>
+   <img class="project-img" alt="project pic" src="https://schemazone.com/wp-content/uploads/2021/03/Project-Manager.jpg">
+ </div>
+</section>
+`;
+});
+
+const modal = document.querySelector('#Work-card-popup');
+for (let i = 0; i < worksArr.length; i += 1) {
+  modal.innerHTML = `<div class="popup">
+    <div class="first-block">
+       <h2 class="work-card-title">${worksArr[i].Title}</h2>
+       <p class="x-icon">${worksArr[i].exitButton}</p>
+     </div>
+     <div class="images-container">
+    <div class="PI"></div>
+    <div class="other-works">
+      <div class="first-project"></div>
+      <div class="first-project"></div>
+      <div class="first-project"></div>
+      <div class="first-project"></div>
+    </div>
+    </div>
+    <span onclick="function next()"class=""><i class="bi bi-chevron-compact-right"></i></span>
+    <span onclick="function prev()"class=""><i class="bi bi-chevron-compact-left"></i></span>
+     <p class="workText">${worksArr[i].firstText}</p>
+     <p class="workText2">${worksArr[i].secondText}</p>
+
+          <ul class="project-tags">
+             <li class="work-tag">${worksArr[i].Skills}</li>
+           </ul>
+       <div class="btns">
+          <button type="button" onclick="" class="btn1">
+            ${worksArr[i].liveDemo}
+            <i class="bi bi-arrow-up-right-circle"></i>
+          </button>
+          <button type="button" onclick="" class="btn2">
+            ${worksArr[i].DemoSource}
+            <img src="images/git-icon.svg" class="icons">
+          </button>
+         </div>
+         <div class="go-to">
+            <span onclick="function prev()"class="previous">${worksArr[i].previousProject}</span>
+            <span onclick="function next()" class="next">${worksArr[i].nextProject}</span>
+         </div>
+         </div>
+`;
+}
+
+const img1 = document.querySelectorAll('first-project')[1];
+const img2 = document.querySelectorAll('first-project')[2];
+const img3 = document.querySelectorAll('first-project')[3];
+const img4 = document.querySelectorAll('first-project')[4];
+const ImgArr = [img1, img2, img3, img4];
+let i = 0;
+
+function setImg() {
+  ImgArr[i + 1].style.border = '1px black';
+}
+
+function prev() {
+  if (i <= 0) i = ImgArr.length;
+  i -= 1;
+  return setImg();
+}
```

Comment:
- [ ] [OPTIONAL] Good implementation your code is clean :+1: I noticed you used the arrows functions for some parts. I would like to recommend you change all your functions to the arrow function to make your code uniform :smile: